src/api/cvc4cpp.cpp

```diff
@@ -4642,6 +4642,15 @@ void Solver::pop(uint32_t nscopes) const
   CVC4_API_SOLVER_TRY_CATCH_END;
 }
 
+bool Solver::getInterpolant(Term B, Term& output) const {
```

Comment:
This should be accompanied by a unit test in `test/unit/api/solver_black.h`.

Reply:
Definitely, but I think only in the next PR, this PR does not include implementations.

Reply:
Right, so either we can add a stub in the test as to not forget about it, or we need a way to keep track of it. I'd prefer the former if possible.